README.md

```diff
@@ -21,22 +21,26 @@ The example below initializes a web project with `launchpad init`, and deploys t
 
 ## Benefits
 
-### Build publish and deploy with a single command
+### Build, publish, and deploy with a single command
 
-Launchpad can build any image, publish it to your Docker registry, and deploy it to Kubernetes in one step.
+No need to remember which container image you uploaded to where, and what kube-context to use, Launchpad can build any image, publish it to your Docker registry, and deploy it to Kubernetes in one step.
```

Comment:
I think we should flip the clauses. The first thing I should read is a clear explanation of what Launchpad does: 

> Launchpad builds any image, publishes it to your Docker Registry, and deploys it to Kubernetes in one step. No need to manually build and push your image, setup your kube-context, or write long pages of Kubernetes YAML.